backend/app/services/word_comment_service.py

```
@@ -0,0 +1,459 @@
+                para_idx = self._find_paragraph_index_for_item(
+                    paragraphs, item, chapter_targets
+                )
+                if para_idx is not None:
+                    comments_to_add.append((para_idx, comment_text))
+
+        # 合规性批注
+        if compliance:
+            items = compliance.get("items", [])
+            for item in items:
+                comment_text = self._format_compliance_comment(item)
+                # 合规性批注：尝试在文档开头或封面区域添加
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)